daprieu/rare-client-cool-cake
# Cr categories view

# Description

User can view categories alphabetically.

## Type of change


# Testing Instructions

`git fetch --all`
`git checkout cr-categories-view`
`npm start`

For server-side instructions, see backend pull request.

After the backend is up and running, in Postman, make a `GET` request to 

`http://localhost:8000/categories`

to view all categories. In browser, after successful login, click on `Category Management` to view a list of categories listed alphabetically.


# Checklist:
